<?xml version="1.0" encoding="UTF-8"?>
<bajadoc version="2.0" createdBy="niagara-baja-doclet-1.0.9" createdAt="03-May-2024" createdOn="883e7f7a9875">
<class module="bajaui" runtimeProfile="wb" qualifiedName="javax.baja.ui.UnboundException" name="UnboundException" packageName="javax.baja.ui" public="true" category="exception">
<description>
UnboundException is thrown when attempting to access &#xa; the target of an unbound BBinding.
</description>
<tag name="@author">Brian Frank</tag>
<tag name="@creation">17 May 04</tag>
<tag name="@version">$Revision: 2$ $Date: 12/22/05 10:22:56 AM EST$</tag>
<tag name="@since">Baja 1.0</tag>
<extends>
<type class="javax.baja.sys.BajaRuntimeException"/>
</extends>
<!-- javax.baja.ui.UnboundException(java.lang.String, java.lang.Throwable) -->
<constructor name="UnboundException" public="true">
<parameter name="msg">
<type class="java.lang.String"/>
</parameter>
<parameter name="cause">
<type class="java.lang.Throwable"/>
</parameter>
<description>
Construct with message and root cause.
</description>
</constructor>

<!-- javax.baja.ui.UnboundException(java.lang.String) -->
<constructor name="UnboundException" public="true">
<parameter name="msg">
<type class="java.lang.String"/>
</parameter>
<description>
Construct with message.
</description>
</constructor>

<!-- javax.baja.ui.UnboundException() -->
<constructor name="UnboundException" public="true">
<description>
No arg constructor.
</description>
</constructor>

</class>
</bajadoc>
